hathor/p2p/states/ready.py

```diff
@@ -30,10 +32,47 @@
 
 logger = get_logger()
 
+settings = HathorSettings()
+
+
+class BlockInfo(NamedTuple):
+    hash_hex: str
+    height: int
+    weight: float
+
+    @staticmethod
+    def from_raw(block_info_raw):
+        """ Instantiate BlockInfo from a string raw.
```

Comment:
I feel this docstring is wrong since `block_info_raw` must be a `tuple[str, int, float]`.

Reply:
Thanks!